app/models/restaurant.rb

```diff
@@ -28,4 +28,8 @@ class Restaurant < ApplicationRecord
   has_many :reviews
 
   validates_associated :cuisine
+
+  def rating
+    reviews.average(:rating)
```

Comment:
did you check what happens when the restaurant has no reviews? does this method handle this situation properly?

Reply:
returns nil